### PR TITLE
Add Dec 2021 cloud updates

### LINKED ISSUE #3864: Telegraf configuration docs missing reference to exclude_bucket_tag setting
The Telegraf configuration documents do not mention the setting exclude_bucket_tag, which handily prevents the bucket tag (used for routing to the correct bucket) from being unnecessarily added to the output.

##### Relevant URLs
- https://docs.influxdata.com/telegraf/v1.21/administration/configuration/
- _Provide other relevant URLs (project issues, community threads, etc.)_

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #3864

### CODE REVIEW

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ To find information about the latest Flux updates in InfluxDB Cloud, see [Flux r
 
 To enhance security, the Token UI will only display an InfluxDB Cloud token when it's first created. If you return to the Token page later, you will not be able to view or copy the token. -->
 
+## December 2021
+
+- **Paginated tasks in UI**: Previously, the Dashboards page only listed the first 100 dashboards. Now, all tasks are accessible and paginated on the Dashboards page.
```
Comment (noramullen1) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:24]: Assuming this works the same as the task pagination? let me know if not.

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ To find information about the latest Flux updates in InfluxDB Cloud, see [Flux r
...
+- **$250 usage credit**: <!-- is this something we want to share here? -->
+
```
Comment (noramullen1) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:26]: Is there anything else we should add here?

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md (hunk 1 above)
Comment (kelseiv) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:24]:
```suggestion
- **Paginated dashboards in UI**: Previously, the Dashboards page could take awhile to load with more than a few dashboards. Now, all dashboarrds are  immediately paginated and accessible on the Dashboards page.
```

**Comment by noramullen1** [comment]
`!signed-cla`

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 3]
```diff
@@ -19,6 +19,15 @@ To find information about the latest Flux updates in InfluxDB Cloud, see [Flux r
 
 To enhance security, the Token UI will only display an InfluxDB Cloud token when it's first created. If you return to the Token page later, you will not be able to view or copy the token. -->
 
+## December 2021
+
+- **Paginated dashboards in UI**: Previously, the Dashboards could take a while to load with more than a few dashboards. Now, all tasks are immediately accessible on the Dashboards page.
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:24]:
```suggestion
- **Paginated dashboards in UI**: Previously, the Dashboards page could take awhile to load with more than a few dashboards. Now, all dashboards are immediately paginated and accessible on the Dashboards page.
```

**Review by kelseiv — APPROVED** [review]

**Pull request merged**